src/pages/sighting/SightingCore.jsx

```diff
@@ -64,6 +66,10 @@ export default function SightingCore({
     onClearError: deleteAsgOnClearError,
   } = useDeleteAssetGroupSighting();
 
+  const onClearError = pending
+    ? deleteAsgOnClearError
+    : deleteSightingOnClearError;
```

Comment:
This should also be used here
https://github.com/WildMeOrg/codex-frontend/blob/d8a689389916b96156f9b4858a2311cabffc4716/src/pages/sighting/SightingCore.jsx#L146-L148

Reply:
Good catch. Done.